pyproject.toml

```diff
@@ -1,3 +1,76 @@
+[build-system]
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "breathe"
+dynamic = ["version"]
+authors = [
+    {name = "Michael Jones", email = "m.pricejones@gmail.com"},
+]
+description = "Sphinx Doxygen renderer"
+license = {text = "BSD-3-Clause"}
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Environment :: Web Environment",
+    "Framework :: Sphinx :: Extension",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Topic :: Documentation",
+    "Topic :: Text Processing",
+    "Topic :: Utilities",
+]
+requires-python = ">=3.7"
+dependencies = [
+    "docutils>=0.12",
+    "Jinja2>=2.7.3",
+    "MarkupSafe>=0.23",
+    "Pygments>=1.6",
+    "Sphinx>=4.0,!=5.0.0",
+]
+
+[project.optional-dependencies]
+lint = [
+    "black==22.3.0",
+    "flake8",
+    "mypy>=0.900",
+]
+test = [
+    "pytest",
+]
+dev = [
+    "breathe[lint,test]",
+    "furo",
```

Comment:
Why not split this into `docs`?

Reply:
yes, there could be a `docs` extra. What would be moved into it? Is it just `sphinx-copybutton` and `furo`?

Reply:
Pretty much that. Also `types` should be bundled together with `lint`. `sphinx` doesn't use it maybe?